.travis/run.sh

```diff
@@ -85,6 +85,60 @@ for BOARD in $BOARDS; do
 				PROG=$PROG BOARD=$BOARD TARGET=$TARGET make load || true
 			done
 		fi
+		
+		# Copy built files
+		if [ -z $GITHUB_TOKEN ]; then
+			echo ""
+			echo ""
+			echo ""
+			echo "- No Github token so unable to copy built files"	
+		else
+			echo ""
+			echo ""
+			echo ""
+			echo "- Fetching non shallow to get git version"
+			echo "---------------------------------------------"
+			git fetch --unshallow && git fetch --tags
+			GIT_REVISION=`git describe`
+			echo "============================================="
+			echo ""
+			echo ""
+			echo ""
+			echo "- Copying built files"
+			echo "---------------------------------------------"
+			COPY_REPO_OWNER="sealne"
+			COPY_REPO="HDMI2USB-firmware-prebuilt"
+			COPY_DEST="archive/$GIT_REVISION/$BOARD/$TARGET"
+			rm -rf $COPY_REPO
+			git clone https://$GITHUB_TOKEN@github.com/$COPY_REPO_OWNER/$COPY_REPO.git			
+			cd $COPY_REPO
+			mkdir -p $COPY_DEST
+			# Not currently built so use .bit instead
```

Comment:
Maybe we should add a "make release" or something which collects all the files into the `build/release` directory or something?